### DISCUSSION TITLE
Publish site by github - recognized as dangerous site

### DISCUSSION THREAD

**Discussion by shokerm** [community/community#32098] [Other Feature Feedback, Questions, & Ideas] [answered]
I published a simple site by github for practice, and it is recognized as dangerous site for sone reason.
What I need to do to fix it?
For Example this >>> https://shokerm.github.io/JS_Events_HS/

**Comment by radar-blip** [comment, marked as answer]
This has happened a few times lately. Check marked answer here: #30261 it should help solve issue.
